apps/web/src/components/hero/index.tsx

```diff
@@ -3,16 +3,38 @@
 import Image from "next/image";
 import { useEffect, useState } from "react";
 import { cn } from "../../lib/utils";
+import { useScramble } from "use-scramble";
 
-export function Hero({ images, text }: { images: string[]; text: string }) {
+export function Hero({ images, texts }: { images: string[]; texts: string[] }) {
   const [currentImage, setCurrentImage] = useState(
     new Date().getUTCDate() % images.length,
   );
 
+  const [currentText, setCurrentText] = useState("");
```

Comment:
Can we make the default text texts[0] or a random text so it shows something on initial server render? Perhaps similarly to images above